src/lib/http-server.ts

```diff
@@ -29,14 +30,20 @@ import { assertIsAddressInfo } from './helpers';
  */
 export class HttpServer {
   #server: Server;
+  sslEnabled = false;
```

Comment:
@lbestftr
- That name may be a little too protocol specific. How about something like `isSecured`?
- Maybe making this private as well?

Reply:
Fixed!